tests/riotboot_flashwrite/Makefile

```diff
@@ -13,40 +33,64 @@ USEMODULE += sock_udp
 # Additional networking modules that can be dropped if not needed
 USEMODULE += gnrc_icmpv6_echo
 
+# Required for nanocoap server
 USEMODULE += nanocoap_sock
 
 # include this for printing IP addresses
+USEMODULE += shell
 USEMODULE += shell_commands
 
-# Comment this out to enable code in RIOT that does safety checking
-# which is not needed in a production environment but helps in the
-# development process:
-#DEVELHELP = 1
+# include riotboot modules
+USEMODULE += riotboot_flashwrite
+FEATURES_REQUIRED += riotboot
+
+# Change this to 0 to not use ethos
+USE_ETHOS ?= 1
 
-# Use different settings when compiling for one of the following (low-memory)
-# boards
-LOW_MEMORY_BOARDS := nucleo-f334r8
+ifeq (1,$(USE_ETHOS))
+  GNRC_NETIF_NUMOF := 2
```

Comment:
This does no longer exist

Reply:
Yes this was an old branch, I rebased but did not re-adapt the style